src/App.jsx

```diff
@@ -1,5 +1,44 @@
+import { useCookies } from 'react-cookie';
+import { useState, useEffect } from 'react';
+import AuthService from './services/AuthService';
+
 function App() {
-  return <div>Hello world!</div>;
+  const [cookies] = useCookies(['token']);
+  const [locations, setLocations] = useState([]);
+
+  useEffect(async () => {
+    const searchedLocations = await getSearchLocation('London', cookies);
+    setLocations(searchedLocations.locations);
+  }, []);
+
+  async function getSearchLocation(location, cookies) {
+    const accessToken = await AuthService.getAccessToken(cookies);
+    const url = `/api/v1/location/search/${location}`;
+
+    const fetchResponse = await fetch(url, {
```

Comment:
it would be better to move fetch method to another folder(like api or else). And here call only method

Reply:
Done. This fetch method was moved to src/services/ApiService.js